docs/usage/docker.md

```diff
@@ -279,12 +279,47 @@ To make use of this authentication mechanism, specify the username as `AWS`:
 
 #### Google Container Registry / Google Artifact Registry
 
-##### Using Application Default Credentials / Workload Identity (Self-Hosted only)
+##### Using Workload Identity
 
-Just configure [ADC](https://cloud.google.com/docs/authentication/provide-credentials-adc) /
-[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) as normal and _don't_
-provide a username, password or token. Renovate will automatically retrieve the credentials using the
-google-auth-library.
+[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) must be configured and the Service Account must have the `artifactregistry.repositories.downloadArtifacts` permission.
+
+###### With Application Default Credentials (Self-Hosted only)
+
+Configure [ADC](https://cloud.google.com/docs/authentication/provide-credentials-adc) as normal.
+Do not provide a username, password or token.
+Renovate will get the credentials with the [`google-auth-library`](https://www.npmjs.com/package/google-auth-library).
+
+###### With short-lived access token / GitHub Actions (Self-hosted only)
+
+Example configuration for Workload Identity and Renovate's host rules:
+
+```yaml
```

Comment:
Please:

1. Delete line 294: `Example configuration for Workload Identity and Renovate's host rules:`.
2. Add a title to the YAML codeblock. [^1]

The bit after the three backticks should look like this:

`yaml title="Example for Workload Identity plus Renovate host rules"`

[^1]: [Material for MkDocs, adding a title to a codeblock](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title)